Server/mods/deathmatch/logic/luadefs/CLuaWorldDefs.cpp

```diff
@@ -797,6 +802,17 @@ int CLuaWorldDefs::getFogDistance(lua_State* luaVM)
     return 1;
 }
 
+std::variant<CLuaMultiReturn<float, float>, bool> CLuaWorldDefs::GetGrassDrawDistance()
+{
+    float fCloseDistance, fFarDistance;
+    bool  bSuccess = CStaticFunctionDefinitions::GetGrassDrawDistance(fCloseDistance, fFarDistance);
+
+    if (bSuccess)
+        return std::make_tuple(fCloseDistance, fFarDistance);
```

Comment:
Maintain consistency with the client-side version of the function and return CLuaMultiReturn